locales/de/generate.json

```diff
@@ -0,0 +1,28 @@
+{
+  "form": {
+    "title": "Kalender generieren",
+    "description": "Zuerst, wähle welche F1 Sessions du zu deinem Kalender hinzufügen willst:",
```

Comment:
Would change the order of words in the sentence a bit:
'Wähle zuerst, welche F1 Sitzungen du zu deinem Kalender hinzufügen willst.'

Reply:
Yes, I'm just not sure with the word 'Sitzungen', but I'm also not satisfied with keeping 'Sessions'. There must be better word.

Reply:
![image](https://user-images.githubusercontent.com/22171518/84570816-0fc9ca00-ad90-11ea-9d4e-95f5457d40a1.png)
session is apparently used frequently in german, according to google translate

Reply:
Ok, then I will stick with Session.